tests/debug/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+#configure_file(common.sh common.sh)
+
+set(DEBUG_TESTS
+    test_xdd_debug_init.sh
+    )
+
+# original tests depended on xdd being installed
+option(TEST_INSTALLED "Use installed xdd for tests" ON)
+if (TEST_INSTALLED)
+  set(XDD_EXEC "$<TARGET_FILE:xdd>")
+else()
+  set(XDD_EXEC "${CMAKE_BINARY_DIR}/bin/xdd")
+endif()
```

Comment:
Duplicate

Reply:
Ughhh... I forgot to cleanup some stuff before creating the PR. I just put another commit on top of the original that cleans up everything. We need to squash this down into the original commit if we decide to merge this.